src/server/game/Entities/AreaTrigger/AreaTrigger.h

```diff
@@ -21,11 +21,14 @@
 #include "Object.h"
 #include "Position.h"
 #include "Spline.h"
+#include "AreaTriggerAI.h"
```

Comment:
you will need to add it in .cpp files that need it (instead of all that include AreaTrigger.h but don't need the AI header)